src/postgraphql/withPostGraphQLContext.js

```diff
@@ -0,0 +1,45 @@
+import { $$pgClient } from '../postgres/inventory/pgClientFromContext'
+import debugPgClient from './http/debugPgClient'
+import setupPgClientTransaction from './setupPgClientTransaction'
+
+/*
+
+This is intended to be called similar to this:
+
+    const result = await withPostGraphQLContext(options, { pgPool, jwtToken }, async context => (
+      await graphql(
+        schema,
+        query,
+        null,
+        {  ...context },
+        variables,
+        operationName,
+      )
+    ))
+
+*/
+
+export default async function withPostGraphQLContext(options, {pgPool, jwtToken}, functionToRun) {
+  // Connect a new Postgres client and start a transaction.
+  const pgClient = await pgPool.connect()
+
+  // Enhance our Postgres client with debugging stuffs.
+  debugPgClient(pgClient)
+
+  // Begin our transaction and set it up.
+  await pgClient.query('begin')
+  await setupPgClientTransaction(jwtToken, pgClient, options)
```

Comment:
Done 👍  I had to disable a bunch of tests because they reference the now removed method, that'll likely have to have something done about but I don't really have time to re-implement all those tests.